cicd/deploy_ephemeral_env.sh

```diff
@@ -42,6 +43,7 @@ function get_pod_logs {
 
 function collect_k8s_artifacts {
     mkdir -p $K8S_ARTIFACTS_DIR
+    [ -f "$K8S_ARTIFACTS_DIR/oc_get_all.yaml" ] && return
```

Comment:
right, but as you can see in slack message, teardown was executed 3 times when I sent SIGTERM, so I wanted to avoid unnecessary collection if that happens